frontend/src/components/guideCarousel/index.js

```diff
@@ -0,0 +1,218 @@
+import Grid from '@mui/material/Grid'
+import { Button } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import Carousel from 'react-material-ui-carousel'
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft'
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+import KeyboardTemplate from './keyboardTemplate'
+import RobotMovement from './activeButtons/robotMovement'
+import RobotMovementObliquely from './activeButtons/robotMovementObliquely'
+import RobotMovementObliquelyHolo from './activeButtons/robotMovementObliquelyHolo'
+import Roll from './activeButtons/roll'
+import Pitch from './activeButtons/pitch'
+import Torsion from './activeButtons/torsion'
+import Tilt from './activeButtons/tilt'
+import TiltSide from './activeButtons/tiltSide'
+import BodyElevetion from './activeButtons/bodyElevetion'
+import robot from '../../Media/pupper_cool.jpeg'
+
+
+const StyledCarousel = styled(Carousel)(({ theme }) => ({
+  '& .nav-button-wrapper': {
+    height: 'auto',
+    top: 'unset',
+    bottom: '-2px',
+    '&:hover button': {
+      backgroundColor: 'transparent',
+      filter: 'unset',
+      opacity: 0.6,
+    }
+  },
+  '& .nav-button': {
+    fontSize: '14px',
+    fontWeight: theme.typography.fontWeightMedium,
+    color: theme.palette.info.main,
+    fontFamily: theme.typography.fontFamily,
+    backgroundColor: 'transparent',
+    textTransform: 'capitalize',
+    transition: 'opacity ease .4s',
+    '& svg': {
+      fontSize: '20px',
+    }
+  },
+  '& .indicator-wrapper': {
+    marginTop: '32px',
+  },
+  '& .indicator': {
+    
+  },
+  '& .indicator-icon': {
+    color: theme.palette.background.guideCarouselIndicator,
+    padding: '6px',
+    fontSize: '12px',
+  },
+  '& .indicator-icon-active': {
+    color: theme.palette.info.main,
+  }
+}))
+
+const guideItems = [
+  {
+    leggedRobotImage: robot,
+    title: 'Robot movement:',
+    activeButtonsDescription: RobotMovement,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['forward', 'backward', 'leftward-rotation', 'rightward-rotation'],
```

Comment:
It probably makes sense to move actions into enum. I see dups and it's easy to make a mistake or forget to update something.